modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-visualizer.adoc
+
+[id="cluster-logging-visualizer-indices_{context}"]
+= Defining Kibana index patterns
+
+An index pattern defines the Elasticsearch indices that you want to visualize. To explore and visualize data in Kibana, you must create an index pattern. 
+
+.Prerequisites
+
+* A user must have the `cluster-admin` role, the `cluster-reader` role, or both roles to list the *infra* and *audit* indices in Kibana. 
++
+For example:
++
+[source,terminal]
+----
+$ oc auth can-i get pods/logs -n default
+----
++
+.Example output
+[source,terminal]
+----
+yes
+----
++
+[NOTE]
+====
+The audit logs are not stored in the internal {product-title} Elasticsearch instance by default. To view the audit logs in Kibana, you must use the Log Forward API to configure a pipeline that uses the `default` output for audit logs.
```

Comment:
The description is correct for the GA LF(v4.6). 

But, there isn't a default output for LogForward (Tech Preview, v4.3,v4.4,v4.5).  To forward the Elasticsearch lanched by CLO(To avoid using the word 'default', I think we can use 'Elasticsearch launched by CLO. @alanconway ).  The LogForwarding CR is like below:
```
apiVersion: logging.openshift.io/v1alpha1
kind: LogForwarding
metadata:
  name: instance
  namespace: openshift-logging
spec:
  outputs:
    - name: clo-es
      type: elasticsearch
      endpoint: 'elasticsearch-cluster.openshift-logging.svc:9200'
      secret:
        name: fluentd
  pipelines:
    - name: app-pipeline
      inputSource: logs.app
      outputRefs:
        - clo-es
    - name: infra-pipeline
      inputSource: logs.infra
      outputRefs:
        - clo-es
    - name: audit-pipeline
      inputSource: logs.audit
      outputRefs:
        - clo-es
```

The audit indices are as below:
```
.audit.2020.09.11
.kibana
.operations.2020.09.11
project.logflatx.156b8402-aaa1-4d6f-80aa-ac7e61a13c1c.2020.09.11
```